charts/gardener/controlplane/charts/application/templates/namespace-gardener-system-public.yaml

```diff
@@ -0,0 +1,8 @@
+# gardener-apiserver usually creates the namespace on start-up, but helm based
+# installations may install simultaneously gardener-apiserver and the role(binding)s
+# in the gardener-system-public namespace, which makes the later to fail
+# because the namespace is not yet created.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gardener-system-public
```

Comment:
There is `kube-system` and `kube-public` -> should we simply call this `gardener-public` then?

Reply:
Okay, fair enough, let's keep `gardener-system-public` then given that we already have other namespaces with this prefix 👍🏻